mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewgroup/CreateNewGroupFragment.java

```diff
@@ -218,12 +219,22 @@ public void inflateSubmissionDate() {
         tv_submissionDate.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                mfDatePicker.show(getActivity().getSupportFragmentManager(), FragmentConstants
-                        .DFRAG_DATE_PICKER);
+                Dialog dialogFrg = mfDatePicker.getDialog();
+                if (dialogFrg != null && dialogFrg.isShowing()) {
+                // no need to call inputSubmiss ionDate()
+                } else {
+                inputSubmissionDate();
```

Comment:
incorrect indentation of input submission date

Reply:
In the comment?

Reply:
Or I have to add a "TAB" before the inputSubmissionDate()

Reply:
tab before the function call

Reply:
okay